# Notebook runs succeccfully but submission error

**Yangtze Hao** *Mon Jul 15 2024 17:59:44 GMT+0900 (日本標準時)* (1 votes)



I have tested the agent in the notebook with the internet off and no error occurred. But the submission keep showing [validation eposide error]. 

The agent 0 log shows that the error occurred after a few successful rounds

To make it clearer, I print the last error output in the log json. It seems that there's something wrong with .generate method of my llm model. But i didn't have any problems running the agent in the notebook. Can anyone help me



